src/project/project.cpp

```diff
@@ -65,28 +110,36 @@ void Project::init(string name, string tpl, bool master) {
   config_template.setValue("job_name", job_name);
 
   FileManager::write(template_path, config_template.render());
+
 }
 
-void Project::create_job(string name, string project_token){
+void Project::existing_init(string name, bool master) {
+  const string source = "project_token: {{token}}\n" 
```

Comment:
Dont we have a template file to pull this from?

Reply:
The user might not. This init can be used without worrying about the update templates command being called. So that base template might not exist.

Reply:
I thought that update template always got called?  If we change the default config then we'll have to remember to update this template

Reply:
yeah having a get_base_template or something would probably be better so we minimize the number of places to update a template